dependencies/common/install-yarn

```diff
@@ -58,5 +58,3 @@ PATH=${INSTALL_PATH} yarn config set ignore-engines true
 PATH=${INSTALL_PATH} yarn install
```

Comment:
nit: INSTALL_PATH is only set when yarn isn't already installed; do we need to set that outside of the if block?

Reply:
I think this matches the old behaviour (though I didn't check if INSTALL_PATH gets set before this block of code), and "it worked when I tried it."

Reply:
Plus when yarn is installed it updates the path (in my case in .zshrc) so on subsequent runs it will be able to find yarn in ~/.yarn/bin.